content/applications/finance/accounting/taxation/taxes/default_taxes.rst

```diff
@@ -2,11 +2,10 @@
 Default taxes
 =============
 
-**Default Taxes** define which :doc:`taxes <taxes>` are automatically selected when there is no
-other indication about which tax to use. For example, Odoo prefills the **Taxes** field with the
-Default Taxes when you create a new product or add a new line on an invoice.
+**Default Taxes** define which :doc:`taxes <taxes>` are automatically selected when creating a new product.
+It is also used to prefill the **Taxes** field with when you add a new line on an invoice in Accounting Firms mode.
 
-.. image:: default_taxes/default-taxes-invoice-line.png
+.. image:: default_taxes/default-taxes-product.png
```

Comment:
For a better resolution, please take the screenshot with 110% zoom (or more if needed). Also, the file should always be compressed, if there are any irrelevant parts, it's better to crop them ( for example, here the internal note field at the bottom can be left out so we can focus on what matters).